jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/GradleRawConfigurationTest.java

```diff
@@ -68,6 +68,7 @@ public void testGetters() {
     Mockito.when(containerParameters.getPorts()).thenReturn(Arrays.asList("80/tcp", "0"));
     Mockito.when(containerParameters.getUseCurrentTimestamp()).thenReturn(true);
     Mockito.when(containerParameters.getUser()).thenReturn("admin:wheel");
+    Mockito.when(containerParameters.getFilesModificationTime()).thenReturn("modification time");
```

Comment:
`"modification time"` seems an odd choice given that the other values returned here are legitimate values.